GRAMMAR

```diff
@@ -33,7 +28,7 @@
 ;; - object (an unordered collection of key value pairs)
 ;; - null
 ;;
-;; Expression types are discussed in the Functions Expressions section.
+;; Expression types are discussed in the [Functions Expressions](#functions-expressions) section.
```

Comment:
I've changed this in our fork for the website. Can we settle on the following spelling instead :
```
[Function Expressions](#function-expressions)
````
I notice this is maybe the only reference to an anchor link. How are those identified? They seem to correspond to titles from the original website.

Reply:
Thanks for catching that, I agree it should be changed.
The anchor links point at headers within the same document, markdown automatically tags all headers with an anchor target. All links should work regardless if they are being viewed within the github wiki or on the website. The website generation code makes any necessary modifications to the links during rendering.